src/app/pages/server-details/server-details.component.scss

```
@@ -68,6 +68,62 @@
   max-width: 100% !important;
 }
 
+:host ::ng-deep .server-description-card {
```

Stylelint (17.13.0)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)